lib/src/index.ts

```diff
@@ -489,8 +489,25 @@ class Valist {
     }
   }
 
+  async canRelease(orgName: string, repoName: string, account: string = this.defaultAccount) {
+    try {
+      const isOrgOwner = await this.isOrgOwner(orgName, account);
+      const isOrgAdmin = await this.isOrgAdmin(orgName, account);
+      const isRepoAdmin = await this.isRepoAdmin(orgName, repoName, account);
+      const isRepoDev = await this.isRepoDev(orgName, repoName, account);
+      return isOrgOwner || isOrgAdmin || isRepoAdmin || isRepoDev;
```

Comment:
Looks great! Only recommendation is that we only need to call `isRepoDev` because OrgAdmin, OrgOwners, and RepoAdmins all inherent that role and will return true for the `isRepoDev.`